<!doctype html><html lang=kr class=no-js> <head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta name=description content="NGeneBio (" n-gen-bio&quot;) makes it easy to access your unique genetic information, so you can take control of your health&quot;><meta name=author content=NGeneBio><link href=https://www.ngenebio.com/foundation/nonclinical-studies/ rel=canonical><link rel=alternate href=/en/ hreflang=en><link rel=alternate href=/ hreflang=kr><link rel="shortcut icon" href=../../assets/images/alpha-n-circle.svg><meta name=generator content="mkdocs-1.1.2, mkdocs-material-6.2.6+insiders-1.16.1"><title>Nonclinical Studies - NGeneBio Help Center</title><link rel=stylesheet href=../../assets/stylesheets/main.eb9f5a45.min.css><link rel=stylesheet href=../../assets/stylesheets/palette.697e47cb.min.css><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Noto+Sans+KR:300,400,400i,700%7CMonaco&display=fallback"><style>:root{--md-text-font-family:"Noto Sans KR",;--md-code-font-family:"Monaco",}</style><link rel=stylesheet href=../../stylesheets/extra.css><script>window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)},ga.l=+new Date,ga("create","None","auto"),ga("set","anonymizeIp",!0),ga("send","pageview"),document.addEventListener("DOMContentLoaded",function(){document.forms.search&&document.forms.search.query.addEventListener("blur",function(){if(this.value){var e=document.location.pathname;ga("send","pageview",e+"?q="+this.value)}})}),document.addEventListener("DOMContentSwitch",function(){ga("send","pageview",document.location.pathname)})</script><script async src=https://www.google-analytics.com/analytics.js></script><script>document.addEventListener("DOMContentLoaded",function(){document.body.addEventListener("click",function(e){if(e.target instanceof HTMLElement){var t=e.target.closest("a[href^=http]");t&&ga("send","event","outbound","click",t.href)}})})</script><meta name=robots content=noindex><meta property=og:type content=website><meta property=og:title content="NGeneBio Help Center - Nonclinical Studies"><meta property=og:description content="NGeneBio (" n-gen-bio&quot;) makes it easy to access your unique genetic information, so you can take control of your health&quot;><meta content=https://www.ngenebio.com/foundation/nonclinical-studies/ property=og:url><meta property=og:image content=https://www.ngenebio.com/assets/images/banner.png><meta property=og:image:type content=image/png><meta property=og:image:width content=1080><meta property=og:image:height content=568><meta name=twitter:card content=summary_large_image><meta name=twitter:site content=@squidfunk><meta name=twitter:creator content=@squidfunk><meta name=twitter:title content="NGeneBio Help Center - Nonclinical Studies"><meta name=twitter:description content="NGeneBio (" n-gen-bio&quot;) makes it easy to access your unique genetic information, so you can take control of your health&quot;><meta name=twitter:image content=https://www.ngenebio.com/assets/images/banner.png><link rel=stylesheet href=../../assets/stylesheets/overrides.fc79678c.min.css></head> <body dir=ltr data-md-color-scheme=preference data-md-color-primary=teal data-md-color-accent=blue> <script>var palette=JSON.parse(localStorage.getItem("__palette"));if(null!==palette&&"object"==typeof palette.color)for(var key in palette.color)document.body.setAttribute("data-md-color-"+key,palette.color[key])</script> <input class=md-toggle data-md-toggle=drawer type=checkbox id=__drawer autocomplete=off> <input class=md-toggle data-md-toggle=search type=checkbox id=__search autocomplete=off> <label class=md-overlay for=__drawer></label> <div data-md-component=skip> <a href=#nonclinical-studies class=md-skip> Skip to content </a> </div> <div data-md-component=announce> <aside class=md-announce> <div class="md-announce__inner md-grid md-typeset"> <a href=https://twitter.com/squidfunk> For updates follow <strong>@squidfunk</strong> on <span class="twemoji twitter"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 512 512"><path d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"/></svg> </span> <strong>Twitter</strong> </a> </div> </aside> </div> <header class="md-header md-header--lifted" data-md-component=header> <nav class="md-header__inner md-grid" aria-label=Header> <a href=https://www.ngenebio.com/ title="NGeneBio Help Center" class="md-header__button md-logo" aria-label="NGeneBio Help Center"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M9 7v10h2v-5l2 5h2V7h-2v5l-2-5H9m3-5a10 10 0 0110 10 10 10 0 01-10 10A10 10 0 012 12 10 10 0 0112 2z"/></svg> </a> <label class="md-header__button md-icon" for=__drawer> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2z"/></svg> </label> <div class=md-header__title data-md-component=header-title> <div class=md-header__ellipsis> <div class=md-header__topic> <span class=md-ellipsis> NGeneBio Help Center </span> </div> <div class=md-header__topic data-md-component=header-topic> <span class=md-ellipsis> Nonclinical Studies </span> </div> </div> </div> <div class=md-header__options> <button class="md-header__button md-icon" title="Switch to light mode" aria-label="Switch to light mode" data-md-option=palette data-md-color-scheme=preference data-md-color-primary=teal data-md-color-accent=blue data-md-state=hidden> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M17 7H7a5 5 0 00-5 5 5 5 0 005 5h10a5 5 0 005-5 5 5 0 00-5-5m0 8a3 3 0 01-3-3 3 3 0 013-3 3 3 0 013 3 3 3 0 01-3 3z"/></svg> </button> <button class="md-header__button md-icon" title="Switch to dark mode" aria-label="Switch to dark mode" data-md-option=palette data-md-color-scheme=slate data-md-color-primary=red data-md-color-accent=red data-md-state=hidden> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M7 10a2 2 0 012 2 2 2 0 01-2 2 2 2 0 01-2-2 2 2 0 012-2m10-3a5 5 0 015 5 5 5 0 01-5 5H7a5 5 0 01-5-5 5 5 0 015-5h10M7 9a3 3 0 00-3 3 3 3 0 003 3h10a3 3 0 003-3 3 3 0 00-3-3H7z"/></svg> </button> <div class=md-select> <span class="md-header__button md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M12.87 15.07l-2.54-2.51.03-.03A17.52 17.52 0 0014.07 6H17V4h-7V2H8v2H1v2h11.17C11.5 7.92 10.44 9.75 9 11.35 8.07 10.32 7.3 9.19 6.69 8h-2c.73 1.63 1.73 3.17 2.98 4.56l-5.09 5.02L4 19l5-5 3.11 3.11.76-2.04M18.5 10h-2L12 22h2l1.12-3h4.75L21 22h2l-4.5-12m-2.62 7l1.62-4.33L19.12 17h-3.24z"/></svg> </span> <div class=md-select__inner> <ul class=md-select__list> <li class=md-select__item> <a href=/en/ class=md-select__link> English </a> </li> <li class=md-select__item> <a href=/ class=md-select__link> 한국어 </a> </li> </ul> </div> </div> </div> <label class="md-header__button md-icon" for=__search> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0116 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 019.5 16 6.5 6.5 0 013 9.5 6.5 6.5 0 019.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg> </label> <div class=md-search data-md-component=search role=dialog> <label class=md-search__overlay for=__search></label> <div class=md-search__inner role=search> <form class=md-search__form name=search> <input type=text class=md-search__input name=query aria-label=검색 placeholder=검색 autocapitalize=off autocorrect=off autocomplete=off spellcheck=false data-md-component=search-query data-md-state=active required> <label class="md-search__icon md-icon" for=__search> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0116 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 019.5 16 6.5 6.5 0 013 9.5 6.5 6.5 0 019.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg> </label> <nav class=md-search__options aria-label=Search> <a href=javascript:void(0) class="md-search__icon md-icon" aria-label=Share data-clipboard data-clipboard-text data-md-component=search-share tabindex=-1> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7 0-.24-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 003-3 3 3 0 00-3-3 3 3 0 00-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 00-3 3 3 3 0 003 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91 1.61 0 2.92-1.3 2.92-2.91A2.92 2.92 0 0018 16.08z"/></svg> </a> <button type=reset class="md-search__icon md-icon" aria-label=Clear data-md-component=search-reset tabindex=-1> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"/></svg> </button> </nav> <div class=md-search__suggest data-md-component=search-suggest></div> </form> <div class=md-search__output> <div class=md-search__scrollwrap data-md-scrollfix> <div class=md-search-result data-md-component=search-result> <div class=md-search-result__meta> Initializing search </div> <ol class=md-search-result__list></ol> </div> </div> </div> </div> </div> <div class=md-header__source> <a href=https://github.com/ngenebio/hgvs title="저장소로 이동" class=md-source> <div class="md-source__icon md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 448 512"><path d="M439.55 236.05L244 40.45a28.87 28.87 0 00-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 01-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 000 40.81l195.61 195.6a28.86 28.86 0 0040.8 0l194.69-194.69a28.86 28.86 0 000-40.81z"/></svg> </div> <div class=md-source__repository> ngenebio/hgvs </div> </a> </div> </nav> <nav class=md-tabs aria-label=Tabs data-md-component=tabs> <div class="md-tabs__inner md-grid"> <ul class=md-tabs__list> <li class=md-tabs__item> <a href=../.. class=md-tabs__link> Home </a> </li> <li class=md-tabs__item> <a href=../../panel/ class=md-tabs__link> Panels </a> </li> <li class=md-tabs__item> <a href=../../manual/oncoaccupanel/overview/ class=md-tabs__link> Software Manual </a> </li> <li class=md-tabs__item> <a href=../../v1_5_2_0/ class=md-tabs__link> Software Changelog </a> </li> </ul> </div> </nav> </header> <div class=md-container data-md-component=container> <main class=md-main data-md-component=main> <div class="md-main__inner md-grid"> <div class="md-sidebar md-sidebar--primary" data-md-component=navigation> <div class=md-sidebar__scrollwrap> <div class=md-sidebar__inner> <nav class="md-nav md-nav--primary md-nav--lifted" aria-label=Navigation data-md-level=0> <label class=md-nav__title for=__drawer> <a href=https://www.ngenebio.com/ title="NGeneBio Help Center" class="md-nav__button md-logo" aria-label="NGeneBio Help Center"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M9 7v10h2v-5l2 5h2V7h-2v5l-2-5H9m3-5a10 10 0 0110 10 10 10 0 01-10 10A10 10 0 012 12 10 10 0 0112 2z"/></svg> </a> NGeneBio Help Center </label> <div class=md-nav__source> <a href=https://github.com/ngenebio/hgvs title="저장소로 이동" class=md-source> <div class="md-source__icon md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 448 512"><path d="M439.55 236.05L244 40.45a28.87 28.87 0 00-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 01-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 000 40.81l195.61 195.6a28.86 28.86 0 0040.8 0l194.69-194.69a28.86 28.86 0 000-40.81z"/></svg> </div> <div class=md-source__repository> ngenebio/hgvs </div> </a> </div> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../.. class=md-nav__link> Home </a> </li> <li class="md-nav__item md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=nav-2 type=checkbox id=nav-2> <label class=md-nav__link for=nav-2> Panels <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=Panels data-md-level=1> <label class=md-nav__title for=nav-2> <span class="md-nav__icon md-icon"></span> Panels </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../../panel/ class=md-nav__link> Overview </a> </li> <li class="md-nav__item md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=nav-2-2 type=checkbox id=nav-2-2> <label class=md-nav__link for=nav-2-2> 유방암&난소암 패널 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label="유방암&난소암 패널" data-md-level=2> <label class=md-nav__title for=nav-2-2> <span class="md-nav__icon md-icon"></span> 유방암&난소암 패널 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../../brcaaccutest/ class=md-nav__link> BRCAaccuTest </a> </li> <li class=md-nav__item> <a href=../../brcaaccutest-plus.md class=md-nav__link> BRCAaccuTest Plus </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=nav-2-3 type=checkbox id=nav-2-3> <label class=md-nav__link for=nav-2-3> 혈액암 패널 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label="혈액암 패널" data-md-level=2> <label class=md-nav__title for=nav-2-3> <span class="md-nav__icon md-icon"></span> 혈액암 패널 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../../heme-accutest.md class=md-nav__link> HEMEaccuTest </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=nav-2-4 type=checkbox id=nav-2-4> <label class=md-nav__link for=nav-2-4> 고형암 패널 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label="고형암 패널" data-md-level=2> <label class=md-nav__title for=nav-2-4> <span class="md-nav__icon md-icon"></span> 고형암 패널 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../../solid-accutest.md class=md-nav__link> SOLIDaccuTest </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=nav-2-5 type=checkbox id=nav-2-5> <label class=md-nav__link for=nav-2-5> 희귀유전질환 패널 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label="희귀유전질환 패널" data-md-level=2> <label class=md-nav__title for=nav-2-5> <span class="md-nav__icon md-icon"></span> 희귀유전질환 패널 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../../heread-accutest.md class=md-nav__link> HEREDaccuTest </a> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=nav-2-6 type=checkbox id=nav-2-6> <label class=md-nav__link for=nav-2-6> 조직적합항원 패널 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label="조직적합항원 패널" data-md-level=2> <label class=md-nav__title for=nav-2-6> <span class="md-nav__icon md-icon"></span> 조직적합항원 패널 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../../hla-accutest.md class=md-nav__link> HLAaccuTest </a> </li> </ul> </nav> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=nav-3 type=checkbox id=nav-3> <label class=md-nav__link for=nav-3> Software Manual <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label="Software Manual" data-md-level=1> <label class=md-nav__title for=nav-3> <span class="md-nav__icon md-icon"></span> Software Manual </label> <ul class=md-nav__list data-md-scrollfix> <li class="md-nav__item md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=nav-3-1 type=checkbox id=nav-3-1> <label class=md-nav__link for=nav-3-1> ONCOaccuPanel <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=ONCOaccuPanel data-md-level=2> <label class=md-nav__title for=nav-3-1> <span class="md-nav__icon md-icon"></span> ONCOaccuPanel </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../../manual/oncoaccupanel/overview/ class=md-nav__link> Overview </a> </li> <li class=md-nav__item> <a href=../../manual/oncoaccupanel/login/ class=md-nav__link> 로그인 </a> </li> <li class=md-nav__item> <a href=../../manual/oncoaccupanel/request/ class=md-nav__link> 신규분석요청 </a> </li> <li class=md-nav__item> <a href=../../manual/oncoaccupanel/status/ class=md-nav__link> 분석 진행 상태 </a> </li> <li class=md-nav__item> <a href=../../manual/oncoaccupanel/result/ class=md-nav__link> 분석 결과 </a> </li> <li class=md-nav__item> <a href=../../manual/oncoaccupanel/report/ class=md-nav__link> 리포트 </a> </li> <li class=md-nav__item> <a href=../../manual/oncoaccupanel/review/ class=md-nav__link> 리뷰 </a> </li> <li class="md-nav__item md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=nav-3-1-8 type=checkbox id=nav-3-1-8> <label class=md-nav__link for=nav-3-1-8> 부록 <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label=부록 data-md-level=3> <label class=md-nav__title for=nav-3-1-8> <span class="md-nav__icon md-icon"></span> 부록 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../../manual/oncoaccupanel/cancer-type/ class=md-nav__link> Cancer type </a> </li> </ul> </nav> </li> </ul> </nav> </li> </ul> </nav> </li> <li class="md-nav__item md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=nav-4 type=checkbox id=nav-4> <label class=md-nav__link for=nav-4> Software Changelog <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label="Software Changelog" data-md-level=1> <label class=md-nav__title for=nav-4> <span class="md-nav__icon md-icon"></span> Software Changelog </label> <ul class=md-nav__list data-md-scrollfix> <li class="md-nav__item md-nav__item--nested"> <input class="md-nav__toggle md-toggle" data-md-toggle=nav-4-1 type=checkbox id=nav-4-1> <label class=md-nav__link for=nav-4-1> Release Note <span class="md-nav__icon md-icon"></span> </label> <nav class=md-nav aria-label="Release Note" data-md-level=2> <label class=md-nav__title for=nav-4-1> <span class="md-nav__icon md-icon"></span> Release Note </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=../../v1_5_2_0/ class=md-nav__link> v1.5.2 </a> </li> <li class=md-nav__item> <a href=../../v1_5_1_0/ class=md-nav__link> v1.5.1 </a> </li> <li class=md-nav__item> <a href=../../v1_5_0_0/ class=md-nav__link> v1.5.0 </a> </li> <li class=md-nav__item> <a href=../../v1_4_4_0/ class=md-nav__link> v1.4.4 </a> </li> <li class=md-nav__item> <a href=../../v1_4_3_0/ class=md-nav__link> v1.4.3 </a> </li> <li class=md-nav__item> <a href=../../v1_4_2_0/ class=md-nav__link> v1.4.2 </a> </li> <li class=md-nav__item> <a href=../../v1_4_1_0/ class=md-nav__link> v1.4.1 </a> </li> <li class=md-nav__item> <a href=../../v1_4_0_0/ class=md-nav__link> v1.4.0 </a> </li> <li class=md-nav__item> <a href=../../v1_3_3_0/ class=md-nav__link> v1.3.0 </a> </li> <li class=md-nav__item> <a href=../../v1_2_0_0/ class=md-nav__link> v1.2.0 </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=../../future/ class=md-nav__link> Future </a> </li> </ul> </nav> </li> </ul> </nav> </div> </div> </div> <div class="md-sidebar md-sidebar--secondary" data-md-component=toc> <div class=md-sidebar__scrollwrap> <div class=md-sidebar__inner> <nav class="md-nav md-nav--secondary" aria-label=목차> <label class=md-nav__title for=__toc> <span class="md-nav__icon md-icon"></span> 목차 </label> <ul class=md-nav__list data-md-scrollfix> <li class=md-nav__item> <a href=#_1 class=md-nav__link> 분석 정확도/일치도 </a> <nav class=md-nav aria-label="분석 정확도/일치도"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#_2 class=md-nav__link> 기존 검사법 비교 </a> </li> <li class=md-nav__item> <a href=#ntrk123 class=md-nav__link> 기존 NTRK1/2/3 융합 검사 비교 </a> </li> <li class=md-nav__item> <a href=#foundationone-ldt class=md-nav__link> FoundationOne LDT와의 비교 </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#_3 class=md-nav__link> 분석 민감도 </a> <nav class=md-nav aria-label="분석 민감도"> <ul class=md-nav__list> <li class=md-nav__item> <a href=#lod class=md-nav__link> 최소검출한계 (LoD) </a> </li> <li class=md-nav__item> <a href=#lod_1 class=md-nav__link> 최소검출한계 (LoD) </a> </li> <li class=md-nav__item> <a href=#limit-of-blank-lob class=md-nav__link> Limit of Blank (LoB) </a> </li> <li class=md-nav__item> <a href=#analytical-sensitivity-tumor-purity class=md-nav__link> Analytical Sensitivity – Tumor Purity </a> </li> </ul> </nav> </li> <li class=md-nav__item> <a href=#_4 class=md-nav__link> 분석 특이도 </a> </li> <li class=md-nav__item> <a href=#carryovercross-contamination class=md-nav__link> Carryover/Cross-Contamination </a> </li> <li class=md-nav__item> <a href=#precision-and-reproducibility class=md-nav__link> Precision and Reproducibility </a> </li> <li class=md-nav__item> <a href=#ntrk-qc-metric class=md-nav__link> NTRK 변이 QC metric </a> </li> <li class=md-nav__item> <a href=#ntrk-bait-set-in-silico class=md-nav__link> NTRK Bait set 커버리지의 In-silico 평가 </a> </li> <li class=md-nav__item> <a href=#reagent-lot class=md-nav__link> Reagent Lot 교환성 </a> </li> <li class=md-nav__item> <a href=#stability class=md-nav__link> Stability </a> </li> <li class=md-nav__item> <a href=#_5 class=md-nav__link> 일반 실험실 장비 및 시약 평가 </a> </li> <li class=md-nav__item> <a href=#guard-bandingrobustness class=md-nav__link> Guard banding/Robustness </a> </li> <li class=md-nav__item> <a href=#tissue-comparability class=md-nav__link> Tissue Comparability </a> </li> <li class=md-nav__item> <a href=#ntrk-in-silico class=md-nav__link> NTRK 변이 탐지를 위한 In-silico 평가 </a> </li> </ul> </nav> </div> </div> </div> <div class=md-content> <article class="md-content__inner md-typeset"> <h1 id=nonclinical-studies>Nonclinical Studies<a class=headerlink href=#nonclinical-studies title="Permanent link">&para;</a></h1> <p>성능 특성은 광범위한 FFPE 조직 유형에서 파생된 DNA를 사용하여 확립되었다. CDx 표시와 관련된 조직 유형이 각 연구에 포함되었다. 각 연구는 CDx 변형뿐만 아니라 다양한 유전자에 걸친 다양한 유전적 맥락에서 광범위한 대표적인 변형 유형(대체, 삽입-삭제, 복사 번호 변경, 재배열)을 포함했다. MSI와 TMB를 포함한 게놈 시그니처 분석도 실시되었다.</p> <h2 id=_1>분석 정확도/일치도<a class=headerlink href=#_1 title="Permanent link">&para;</a></h2> <h3 id=_2>기존 검사법 비교<a class=headerlink href=#_2 title="Permanent link">&para;</a></h3> <ul> <li>F1CDx에 의한 alteration 검출을 외부 검증 NGS 검사 (evNGS) 결과와 비교</li> <li>46개의 종양 조직에서 추출한 188개의 샘플 포함한 기존 검사법으로 검출된 Short alteration (base substitutions, short indels) 비교 </li> <li>표 6에 PPA와 NPA에 대한 요약 내용</li> <li>전체적으로 F1CDx 검사와 기존 검사 사이에 157개의 중복 유전자가 있음.</li> <li>플랫폼간의 차이는 VUS 변이 차이로 F1CDx와 비교 NGS간에 채택된 필터링의 차이로 예상되며,</li> <li>VUS 필터와 관련 없는 불일치는 주로 homopolymer region의 low alleic fraction을 가진 deletion</li> <li>F1CDx의 variant calling 파이프라인은 homopolymer region의 indel에 대해서 MAF of ≥ 0.10%의 필터를 통해 artifacts에서 비롯되는 false positive를 줄인다.</li> <li>따라서, 관찰된 차이는 두 플랫폼 간의 필터 thresholds가 다르기 때문이다. </li> </ul> <h3 id=ntrk123>기존 NTRK&frac12;/3 융합 검사 비교<a class=headerlink href=#ntrk123 title="Permanent link">&para;</a></h3> <ul> <li>Analytical accuracy를 평가하기 위해 F1CDx와 외부 검증 NGS 검사 (evNGS) 의 NTRK&frac12;/3 rearragement 비교</li> <li>VITRAKVI (laroterctinib)을 위한 LOXO-TRK - 14001 (NCT02122913), -15002 (NAVIGATE, NCT02576431), and -15003 (SCOUT, NCT02637687)의 FFPE 잔여 38 검체 사용</li> <li>38 검체의 구성은 clinical studies LOXO-TRK - 14001, - 15002, and -15003 included the following cancer types으로 구성</li> <li>암종은 bone sarcoma (1), breast (1), cholangiocarcinoma (2), colorectal cancer (7), infantile fibrosarcoma (2), lung (4), melanoma (1), oral cancer (1), primary CNS (1), salivary gland (3), soft tissue sarcoma (10), neuroendocrine (2), thyroid (2), and urinary (1) cancer.</li> <li>제한된 임상시험 샘플로 인해 이전 FMI에서 평가되었된 FMI 임상 기록 보관소 (FMI clinical archives)의 NTRK&frac12;/3 재배열 양성 (N=94) 샘플 포함</li> <li>해당 94 샘플은 breast (14), cholagiocarcinoma (1), colorectal cancer (15), gastrointestinal stromal tumor GIST (2), head and neck (14), lung (10), neuroendocrine (1), reproductive (11), salivary gland (9), thyroid (6), and urinary (6) cancer 와 암종을 알 수 없은 5검체 포함</li> <li>이전 일치성 검증 연구에서 exNGS의 NTRK&frac12;/3 재배열 음성 샘플 포함</li> <li>이전 일치 연구는 NTRK&frac12;/3 재배열 유무에 대해 새로 평가되어 이전의 일치성 분석에서 검사한 474개의 검체가 포함</li> <li>의도된 사용 모집단과 더 일치하도록 NTRK&frac12;/3 재배열 음성 샘플 세트의 일부로 10개 소아용 샘플 (bone sarcoma (1), GIST (1), head and neck (1), pancreas (1), soft tissue sarcoma (1), and urinary cancer (1) 와 1개의 알수 없는 암 유형 포함)</li> <li>본 분석 정확도 연구에서 고형암 환자의 임상 FFPE 검체 (543 NTRK 음성, 83개 양성 검체) 총 626개플 평가</li> <li>F1CDx 또는 evNGS에서 잘못된 결과가 없습니다. 일치성을 평가하기 위해 두 가지 분석이 수행되었다.</li> </ul> <p><a href=../../assets/screenshots/NTRK_Samples.png><img alt=NTRK_Samples src=../../assets/screenshots/NTRK_Samples.png width=60%></a></p> <ul> <li> <p>1차 분석</p> <ul> <li>일차 분석은 F1CDx 및 evNGS 검사에 의한 NTRK&frac12;/3 재배열 검출의 일치성에 초점</li> <li>샘플의 출처에 따라 2개의 서브셋으로 분류 (588 samples from the FMI clinical archives) 하여 PPA, NPA 계산을 위해 별도 평가</li> <li>사용 모집단에서 NTRK&frac12;/3의 유병률(0.32%)에 따라 조정</li> <li>전체 데이터세트 (n=626) 두 부분 집합을 결합하여 10,000회 부스트래핑하여 weighted PPA 90.00 (75.00%, 100.00%), weighted NPA 99.94% (99.92%, 99.97%)</li> </ul> <p><a href=../../assets/screenshots/NTRK_PPA.png><img alt=NTRK_Samples src=../../assets/screenshots/NTRK_PPA.png width=60%></a></p> <div class="admonition warning"> <p class=admonition-title>Bootstrapping</p> <p>Bootstrapping?</p> </div> <ul> <li>일차 분석에 기반한 불일치 검체는 모두 20개로 18개에서 F1CDx 검사에서 양성인 반면 evNGS 검사에서는 재배열이 감지 되지 않았으며, 나머지 2개 검체에서 evNGS 검사에서는 검출되었지만, F1CDx 검사에서는 검출되지 않음<ul> <li>9 샘플 had breakpoints residing outside of evNGS baited regions</li> <li>5 샘플 had the event observed but were not reported due to being below the evNGS assay limit of detection</li> <li>1 샘플 had an intragenic NTRK3-NTRK3 rearrangement that was intentionally not reported by evNGS duto to being intragenic</li> <li>2 샘플 were not reported by evNGS likely due to low sample quality as reported by the evNGS assay</li> <li>1 샘플 (NTRK3-KDM2A) was not detected by evNGS</li> </ul> </li> </ul> </li> <li> <p>2차 분석</p> <ul> <li>2차 분석은 NTRK&frac12;/3 rearrangement의 CDx 바이오마커에 대한 규칙에 따라 NTRK&frac12;/3 재결합 럼출의 일치성에 초점</li> <li>2차 분석에서는 샘플이 NTRK&frac12;/3 바이오마커 규칙을 만족하는 경우에만 F1CDx 양성으로 간주</li> </ul> <p><a href=../../assets/screenshots/NTRK_PPA_2.png><img alt=NTRK_PPA_2 src=../../assets/screenshots/NTRK_PPA_2.png width=60%></a></p> <ul> <li>2차 분석에서의 불일치는 17개 샘플로 13개는 F1CDx 바이오마커 음성이지만, evNGS 바이오마커 양성</li> <li>13개의 샘플 중 10개는 non-fusion rearragnement로 F1CDx 바이오마커 규칙에 따라 F1CDx에서 음성으로 간주</li> <li>그러나 이 10개는 evNGS 검사가 fusion과 rearragement를 구분할 수 없기 때문에 evNGS 바이오마커 양성으로 간주되어 따라서 이 10개 샘플은 1차 분석에서는 불일치 하는 것으로 간주되지 않음</li> <li>evNGS 검사에서 양성이었지만, F1CDx에서 음성으로 나온 3개의 임상시험 샘플은 VITRAKVI에 반응하지 않은 임상시험에 등록된 환자에서 추출(두명의 환자는 안정된 질병을 가지고 있었고 1명은 진행성 질병을 가지고 있음)</li> </ul> <div class="admonition warning"> <p class=admonition-title>Fusion &amp; rearagement</p> <p>Fusion &amp; Rearragement?</p> </div> </li> </ul> <h3 id=foundationone-ldt>FoundationOne LDT와의 비교<a class=headerlink href=#foundationone-ldt title="Permanent link">&para;</a></h3> <ul> <li>F1 LDT를 사용하여 생성된 후향 데이터를 F1 CDx에 대한 보충 및 지원 데이터로 사용할 수 있도록 F1 CDx 검사에 대한 일치성 연구 수행</li> <li>두 검사를 모두 수행한 165 검체를 가지고 진행</li> <li>Tumor content가 20%-90%의 조직에서 추출한 35개의 서로 다른 tumor types에서 추출한 DNA를 포함</li> <li>Short variants, Substitutions, Indels, CNAs, Amplification, Losses, Rearragements에 대해 비교</li> <li>MSI와 TMB에 대해서도 비교</li> </ul> <p><a href=../../assets/screenshots/ppa_f1cdx.png><img alt=ppa_f1cdx src=../../assets/screenshots/ppa_f1cdx.png width=80%></a></p> <div class="admonition warning"> <p class=admonition-title>양성일치율</p> <ul> <li>양성 일치율 (Positive percent agreement, PPA): 양성 결과를 보이는 검사법이 대조 검사법과 양성 검사 결과가 일치하는 비율<ul> <li>PPA of All short variants = 100xa/(a+c) = 100*1282/(1282+73) = 94.6%</li> <li>PPA of Substitutions = 96.6%</li> <li>PPA of Indels = 83.4%</li> </ul> </li> <li>음성 일치율 (Negative percent agreement, NPA): 음성 결과를 보이는 검사법이 대조 검사법과 음성 결과가 일치하는 비율</li> </ul> <p><a href=../../assets/screenshots/ppa.png><img alt=ppa src=../../assets/screenshots/ppa.png width=55%></a></p> </div> <h2 id=_3>분석 민감도<a class=headerlink href=#_3 title="Permanent link">&para;</a></h2> <h3 id=lod>최소검출한계 (LoD)<a class=headerlink href=#lod title="Permanent link">&para;</a></h3> <ul> <li>EGFR exon 21 L858R, EGFR Exon 20 T790M, EGFR exon 19 deletion, KRAS codons 12/13 substitution and BRAF V600E 의 5개 CDx 바이오마커에 대해 테스트</li> <li>BRCA&frac12;의 alteration in non-repetitive regions or homopolymer repeats &lt; 4 bp alteration in a homopolymer region &gt; 4bp (P160018 참고)</li> <li>각 variant 카테고리당 하나의 FFPE tumor 샘플 선택</li> <li>각 샘플당 6개의 레벨의 MAF에 대해 13회 반복 실험 (샘플당 78회) </li> <li>CLSI EP17-A2에서 제안된 반복실험 횟수를 충족하기 위해 단일 FFPE 샘플과 뱅킹된 DNA 샘플에서 여러개의 서로 다른 DNA 풀링</li> <li>Indels은 그룹화 (homopolymer repeat context)</li> <li>Indels의 범위는 1bp - 42bp insertion, deletion은 최대 276bp</li> <li>homopolymer 반복 지역의 indels은 반복 길이에 따라 더 높은 LoD</li> </ul> <h3 id=lod_1>최소검출한계 (LoD)<a class=headerlink href=#lod_1 title="Permanent link">&para;</a></h3> <ul> <li>다양한 암종의 7개의 샘플을 통해 F1CDx LoD 수행</li> </ul> <p><a href=../../assets/screenshots/ntrk_lod.png><img alt=ppa src=../../assets/screenshots/ntrk_lod.png width=55%></a></p> <ul> <li>각 샘플의 LoD를 평가하기 위해 5개의 타겟 computational tumor purity levels (2.5%, 5%, 10%, 15%, 20%)</li> <li>Computational tumor purity는 genome-wide copy number, tumor ploidy, purity를 예측하는 통계 모델에 관찰된 log-ratio, minor allele frequency 데이터를 fitting하여 계산</li> <li>The log-ratio profile is obtained by normalizing aligned tumor sequence reads by dividing read depth by that of a process-matched normal control, followed by a GC-content bias correction using Loess regression. </li> <li>The minor allele frequency profile is obtained from the heterozygous genome-wide SNPs.</li> <li>Twenty (20) replicates were assessed for each dilution level, except 20%, which examined fourteen (14) replicates. Ninety-four (94) replicates were tested per sample. Each specimen was evaluated close to the minimum input requirements of the assay (50 ng), representing the most challenging evaluation of rearrangement detection.</li> <li> <p>LoDs were determined based on tumor purity and reads using the empirical hit rate method for NTRK1, NTRK2, and NTRK3</p> <ul> <li>Empirical hit rate approach was used when there were less than three hit rate levels. The LoD is defined as the lowest level (tumor purity or reads) with at least 95% detection.</li> <li>A summary of the LoD results based on reads and tumor purity using the empirical hit rate method for NTRK gene alteration in each sample evaluated are shown in Table 22.</li> </ul> </li> </ul> <p><a href=../../assets/screenshots/ntrk_lod_result.png><img alt=ntrk_lod_result src=../../assets/screenshots/ntrk_lod_result.png width=55%></a></p> <ul> <li>The final LoD for each NTRK gene fusion presented was determined as the highest LoD observed per gene. The LoD for each gene are:<ul> <li>NTRK1 fusion: 12.10% tumor purity and 24.55 for chimeric reads</li> <li>NTRK2 fusion: 11.50% tumor purity and 24.16 for chimeric reads</li> <li>NTRK3 fusion: 6.10 % tumor purity and 14.65 for chimeric reads</li> </ul> </li> <li>Given the limited number of samples with NTRK1 and NTRK2 fusions evaluated to determine the assay LoD, a post-market study is planned with additional NTRK1 and NTRK2 fusions samples</li> </ul> <h3 id=limit-of-blank-lob>Limit of Blank (LoB)<a class=headerlink href=#limit-of-blank-lob title="Permanent link">&para;</a></h3> <h3 id=analytical-sensitivity-tumor-purity>Analytical Sensitivity – Tumor Purity<a class=headerlink href=#analytical-sensitivity-tumor-purity title="Permanent link">&para;</a></h3> <ul> <li>F1CDx의 견고성(robustness)을 지원하는데 필요한 최소 종양 분율(tumor fraction) 평가</li> <li>6개의 레벨의 tumor content에 레벨당 13회 반복 총 78 per sample</li> <li>ALK fusion과 ERBB2 amplification의 CDx 바이오마커의 분석 민감도 결정</li> </ul> <div class="admonition warning"> <p class=admonition-title>CLSI EP17-A2</p> <p>CLSI EP17-A2 : Clinical Laboratory Standards Institute의 검출단계별 강도 측정 가이드</p> </div> <div class="admonition warning"> <p class=admonition-title>공시료</p> <p>공시료 (blank sample)란 측정하고자 하는 물질이 실험시료에 포함되지 않은것을 의미</p> </div> <h2 id=_4>분석 특이도<a class=headerlink href=#_4 title="Permanent link">&para;</a></h2> <ul> <li>간섭 물질 (Interfering Substances)</li> <li>In silico Analysis – Hybrid Capture Bait Specificity</li> </ul> <h2 id=carryovercross-contamination>Carryover/Cross-Contamination<a class=headerlink href=#carryovercross-contamination title="Permanent link">&para;</a></h2> <h2 id=precision-and-reproducibility>Precision and Reproducibility<a class=headerlink href=#precision-and-reproducibility title="Permanent link">&para;</a></h2> <ul> <li> <p><strong>Intermediate Precision</strong></p> <ul> <li>NTRK&frac12;/3 융합 검출을 위한 F1CDx의 성능 특서을 위해 고형암 환자 7 샘플에 정밀도는 50ng에 가까운 DNA input을 가지고 평가</li> <li>평가 샘플들은 17.2% ~40.1%에 이르는 계산된 종양 purity (computational tumor purity) </li> <li>통과된 샘플의 컷오프는 20% 정도로 정밀 연구에 평가된 샘플에는 F1CDx 검사에 대한 20% 근처의 tumor purity 샘플이 포함</li> <li>샘플의 chimeric read의 평균 8.48~159.04</li> <li>Fold LoD 기반 chimeric read는 0.58 xLoD ~ 10.86 xLoD</li> </ul> </li> </ul> <p><a href=../../assets/screenshots/precision.png><img alt=precision src=../../assets/screenshots/precision.png width=55%></a></p> <ul> <li> <p><strong>Site-to-Site reproducibility</strong></p> <ul> <li>모리스빌 포함하기 위한 재현성 연구는 NTRK 지원하기 위해 수행되지 않음</li> <li>Site-to-site 재현성의 연구 결과가 post-market study로 제공될 예정</li> </ul> </li> </ul> <div class="admonition warning"> <p class=admonition-title>Computational tumor purity</p> <p>Computational tumor purity</p> </div> <h2 id=ntrk-qc-metric>NTRK 변이 QC metric<a class=headerlink href=#ntrk-qc-metric title="Permanent link">&para;</a></h2> <ul> <li>NTRK&frac12;/3 rearragement를 위한 coverage analysis 기반 real-world data</li> <li>F1CDx 승인 후 76,597 샘플을 사용</li> <li>Median exon coverage는 900x 근처로 QC 기준인 250x 이상임</li> </ul> <p><a href=../../assets/screenshots/exon_coverage.png><img alt=exon_coverage src=../../assets/screenshots/exon_coverage.png width=55%></a></p> <ul> <li>Post-DNA extraction QC metrix (LC DNA yield, HC DNA yield, median exon coverage, % target 100x coverage)</li> </ul> <p><a href=../../assets/screenshots/qc_metrix.png><img alt=qc_metrix src=../../assets/screenshots/qc_metrix.png width=55%></a></p> <h2 id=ntrk-bait-set-in-silico>NTRK Bait set 커버리지의 In-silico 평가<a class=headerlink href=#ntrk-bait-set-in-silico title="Permanent link">&para;</a></h2> <ul> <li>DX1 baitset은 NTRK&frac12;/3의 coding 영역, NTRK1 (introns 8,9,10,11), NTRK2 (intron 12)를 커버</li> <li>NTRK1 intron 8과 NTRK2 intron 12는 not fully baited</li> <li>NTRK3 intron은 비록 포함되지 않지만, NTRK3의 흔한 ETV6는 ETV6-NTRK3 fusion (ETV6 introns 5,6)을 위해 baited</li> <li>ETV6의 intron 5는 not fully baited</li> <li>ETV6 intron 4는 not having baits</li> <li>In-silico 분석을 위해 12,464 임상 검체로 NTRK의 커버리지 계산</li> <li>타겟 영역에 평가결과 median exon coverage는 &gt;250x in &gt;99%</li> <li>NTRK1 intron 8에서 부분적으로 coverage 감소를 보였지만 이는 partially baited로 인한 것임</li> </ul> <div class="admonition warning"> <p class=admonition-title>DNA 검사</p> <p>여기서 확실히 bait에 intron을 포함하는 것으로 봐서 DNA 검사임이 확실함 </p> </div> <h2 id=reagent-lot>Reagent Lot 교환성<a class=headerlink href=#reagent-lot title="Permanent link">&para;</a></h2> <h2 id=stability>Stability<a class=headerlink href=#stability title="Permanent link">&para;</a></h2> <ul> <li>Reagent Stability</li> <li>DNA Stabilit</li> <li>FFPE Slide Stability</li> </ul> <h2 id=_5>일반 실험실 장비 및 시약 평가<a class=headerlink href=#_5 title="Permanent link">&para;</a></h2> <h2 id=guard-bandingrobustness>Guard banding/Robustness<a class=headerlink href=#guard-bandingrobustness title="Permanent link">&para;</a></h2> <h2 id=tissue-comparability>Tissue Comparability<a class=headerlink href=#tissue-comparability title="Permanent link">&para;</a></h2> <h2 id=ntrk-in-silico>NTRK 변이 탐지를 위한 In-silico 평가<a class=headerlink href=#ntrk-in-silico title="Permanent link">&para;</a></h2> <p>A study analyzing F1CDx assay data was conducted to provide coverage analysis based on real-world data to support the pan-tumor indication for NTRK&frac12;/3 rearrangements. Since F1CDx’s approval, 76,597 samples have been processed by the DX1 baitset used by F1CDx for the most prevalent NTRK tumor specimen sites listed in Figure 1. An in-silico analysis was performed to present median exon coverage across these tumor specimen sites. Figure 1 and Table 26 show median exon coverage for all tumor specimen sites to be around 900x, which is above the quality control (QC) threshold of 250x.</p> <p>Table 26 summarizes critical post-DNA extraction QC metrics, namely, the mean of LC DNA yield, HC DNA yield, median exon coverage, % target 100X coverage, and the pass rate for the 15 different biopsy specimen sites.</p> <!--<footer class="tx-content__footer md-typeset">
    <a href="/" title="NGeneBio Help Center">
      <hr>
      <span class="twemoji tx-heart">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M14 20.408c-.492.308-.903.546-1.192.709-.153.086-.308.17-.463.252h-.002a.75.75 0 01-.686 0 16.709 16.709 0 01-.465-.252 31.147 31.147 0 01-4.803-3.34C3.8 15.572 1 12.331 1 8.513 1 5.052 3.829 2.5 6.736 2.5 9.03 2.5 10.881 3.726 12 5.605 13.12 3.726 14.97 2.5 17.264 2.5 20.17 2.5 23 5.052 23 8.514c0 3.818-2.801 7.06-5.389 9.262A31.146 31.146 0 0114 20.408z"/></svg>
      </span>
      <hr>
    </a>
  </footer>--> </article> </div> </div> </main> <footer class=md-footer> <div class="md-footer-meta md-typeset"> <div class="md-footer-meta__inner md-grid"> <div class=md-footer-copyright> <div class=md-footer-copyright__highlight> Copyright &copy; 2021 NGeneBio </div> </div> <div class=md-footer-social> <a href=https://github.com/ngenebio target=_blank rel=noopener title=github.com class=md-footer-social__link> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 496 512"><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg> </a> <a href=https://twitter.com/ngenebio target=_blank rel=noopener title=twitter.com class=md-footer-social__link> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 512 512"><path d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"/></svg> </a> <a href=https://linkedin.com/in/ngenebio/ target=_blank rel=noopener title=linkedin.com class=md-footer-social__link> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 448 512"><path d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"/></svg> </a> <a href=https://www.youtube.com/channel/UCPwdgiY5UjyQsz6nlTKBimQ target=_blank rel=noopener title=www.youtube.com class=md-footer-social__link> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 576 512"><path d="M549.655 124.083c-6.281-23.65-24.787-42.276-48.284-48.597C458.781 64 288 64 288 64S117.22 64 74.629 75.486c-23.497 6.322-42.003 24.947-48.284 48.597-11.412 42.867-11.412 132.305-11.412 132.305s0 89.438 11.412 132.305c6.281 23.65 24.787 41.5 48.284 47.821C117.22 448 288 448 288 448s170.78 0 213.371-11.486c23.497-6.321 42.003-24.171 48.284-47.821 11.412-42.867 11.412-132.305 11.412-132.305s0-89.438-11.412-132.305zm-317.51 213.508V175.185l142.739 81.205-142.739 81.201z"/></svg> </a> </div> </div> </div> </footer> </div> <script src=../../assets/javascripts/vendor.2d9b50f1.min.js></script> <script src=../../assets/javascripts/bundle.b7273581.min.js></script><script id=__lang type=application/json>{"clipboard.copy": "\ud074\ub9bd\ubcf4\ub4dc\ub85c \ubcf5\uc0ac", "clipboard.copied": "\ud074\ub9bd\ubcf4\ub4dc\uc5d0 \ubcf5\uc0ac\ub428", "search.config.lang": "en", "search.config.pipeline": " ", "search.config.separator": "[\\s\\-]+", "search.placeholder": "\uac80\uc0c9", "search.result.placeholder": "\uac80\uc0c9\uc5b4\ub97c \uc785\ub825\ud558\uc138\uc694", "search.result.none": "\uac80\uc0c9\uc5b4\uc640 \uc77c\uce58\ud558\ub294 \ubb38\uc11c\uac00 \uc5c6\uc2b5\ub2c8\ub2e4", "search.result.one": "1\uac1c\uc758 \uc77c\uce58\ud558\ub294 \ubb38\uc11c", "search.result.other": "#\uac1c\uc758 \uc77c\uce58\ud558\ub294 \ubb38\uc11c", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.term.missing": "Missing"}</script> <script>
        app = initialize({
          base: "../..",
          features: ["navigation.tabs", "navigation.tabs.sticky", "search.highlight", "search.share", "search.suggest"],
          search: Object.assign({
            worker: "../../assets/javascripts/worker/search.e77c40e2.min.js"
          }, typeof search !== "undefined" && search),
          version: {"method": "mike"}
        })
      </script> </body> </html>